src/sweeper/__main__.py

```diff
@@ -150,5 +157,31 @@ def run_tool(tool):
     return reports
 
 
+def setup_logging(save_report, scheduled):
+    logger = logging.getLogger('sweeper')
+    logger.setLevel(logging.INFO) 
+    
+    formatter = logging.Formatter(
+        fmt='%(levelname)-7s %(asctime)s %(module)10s:%(lineno)5s %(message)s', datefmt='%m-%d %H:%M:%S'
+        )
+
+    #: always set up console_handler
+    console_handler = logging.StreamHandler(stream=sys.stdout)
+    console_handler.setFormatter(formatter)
+
+    #: use log file when report location not provided and when running from scheduled task
+    if scheduled and not save_report:
+        #: get LOG_FILE_PATH from credentials.py
+        log_file = Path(credentials.LOG_FILE_PATH)
+        file_handler = logging.handlers.RotatingFileHandler(log_file, backupCount=10)
+        file_handler.doRollover()
+        file_handler.setFormatter(formatter) 
+    
+    logger.addHandler(console_handler)
+    logger.addHandler(file_handler)
```

Comment:
**question**: is it ok to add a null handler?

Reply:
I'm pretty sure that this line will throw an exception saying something like `NameError: name 'file_handler' is not defined` if scheduled is `False` and `file_handler` is never defined.

Reply:
good catch, that should be up in the `if` block